src/HttpKernel/TestKernel.php

```diff
@@ -166,6 +166,15 @@ public function getKernelDir()
         return $kernelDir;
     }
 
+    public function getProjectDir()
```

Comment:
as this is the same as getKernelDir, should we make this one call getKernelDir? then a using bundle overwriting getKernelDir would also work correctly.

Reply:
I think we need to overwrite the `getProjectDir` else `getProjectDir` will search the folders up to the `composer.json` and thats mostly not what you want in a TestKernel.

Depends on the bc we remove `getKernelDir` or just change it to:

```php
public function getProjectDir()
{
    return $this->getKernelDir();
}
```

Reply:
I didn't the otherway around, as kernel rootdir will be removed in the future when you drop sf <4.x support. So I've called `getProjectDir` in `getKernelDir`. This way we can easily cleanup the old method in the future.

Is it ok like this or should I switch it around?

Reply:
that makes sense to me. its "only" testing, so if a project had to overwrite getKernelDir, they will now notice when they upgrade and have to change their code to overwrite getProjectDir instead.

Reply:
👍 looks good for me!